### PR TITLE
feat: login page without desing [TAS-32]

### PR DESCRIPTION
**Pull request by femacedogomes**

### CODE REVIEW

File: src/app/login/FormComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      validationSchema={SignInValidationSchema}
+      onSubmit={handleFormSubmit}
+    >
+      {({ errors, touched, handleSubmit }) => (
+        <Form onSubmit={handleSubmit} className="flex flex-col gap-3">
+          <Field
```
Comment (pedrohbl03) [inline src/app/login/FormComponent.tsx:52]: Create a FormItem component. 

This component will be contains the following props:

```
errorMessage: string,
label: string,
invalid: boolean
children: React.Node
className: string
htmlFor: string
```

**Review by pedrohbl03 — CHANGES_REQUESTED** [review]
Resolve the comments

File: src/components/atoms/FormItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const FormItem: React.FC<formItemProps> = ({
+  label,
+  invalid,
+  htmlFor,
+  errorMessage,
+  className,
+  children,
+}) => {
+  return (
+    <form>
```
Comment (pedrohbl03) [inline src/components/atoms/FormItem/index.tsx:21]: What this is inside a form? 
Form is a high level component.

Chage this to a div.

Comment (pedrohbl03) [inline src/components/atoms/FormItem/index.tsx:21]: Invalid is a boolean prop. 

use this to change color of border to red: 

example: 


```suggestion
    <div className={invalid && 'border-rose-500'}>
    
```

File: src/components/atoms/FormItem/index.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      {children}
+      <label htmlFor={htmlFor}>{label}</label>
+      <span className={`text-xs text-red-600 ${className}`}>{invalid}{errorMessage && invalid ? errorMessage : ''}</span>
```
Comment (pedrohbl03) [inline src/components/atoms/FormItem/index.tsx:24]:
```suggestion
 errorMessage && (
      <span className={`text-xs text-red-600 ${className}`}>{errorMessage}</span>
     )
```

File: src/components/atoms/FormItem/index.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    </form>
```
Comment (pedrohbl03) [inline src/components/atoms/FormItem/index.tsx:25]:
```suggestion
    </div>
```

**Review by pedrohbl03 — CHANGES_REQUESTED** [review]
Resolve comments

File: src/components/atoms/FormItem/index.tsx (hunk 2 above)
Comment (femacedogomes) [reply src/components/atoms/FormItem/index.tsx:21]: i got it. thanks

File: src/app/login/FormComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      validationSchema={SignInValidationSchema}
+      onSubmit={handleFormSubmit}
+    >
+      {({ errors, touched, handleSubmit }) => (
+        <Form onSubmit={handleSubmit} className="flex flex-col gap-3">
+          <FormItem errorMessage={errors.email} invalid={touched.email} >
```
[the remark this answers is not in the document]
Comment (femacedogomes) [reply src/app/login/FormComponent.tsx:52]: i got it. I made this opetarion directly in the dumb component.

[the remark this answers is not in the document]
Comment (femacedogomes) [reply src/app/login/FormComponent.tsx:52]: so we dont need to do this operation every time we call it.

**Review by femacedogomes — COMMENTED** [review]
pedro corrections maded

**Comment by femacedogomes** [comment]
This PR will be closed

**Pull request closed without merging**